jest.config.js

```diff
@@ -1,9 +1,9 @@
 /** @type {import('jest').Config} */
 module.exports = {
   transform: {
-    '^.+\\.tsx?$': ['@swc/jest'],
+    '^.+\\.(j|t)sx?$': '@swc/jest',
   },
-  transformIgnorePatterns: ['/node_modules/(?!@openmrs)'],
+  transformIgnorePatterns: ['/node_modules/(?!(@openmrs|uuid))'],
```

Comment:
If we want to handle just the `uuid` package, maybe we'd be better served adding a `moduleNameMapper` entry for it:

```
	  moduleNameMapper: {
			// ...other entries
			'^uuid$': require.resolve('uuid')`
		}
```